reference/docs-conceptual/learn/shell/dynamic-help.md

```diff
@@ -0,0 +1,52 @@
+---
+description: This article explains how to use the dynamic help feature of PSReadLine.
+no-loc: [<kbd>F1</kbd>, <kbd>Alt</kbd>]
+title: Using dynamic help
+ms.date: 7/14/2022
+---
+# Using dynamic help
+
+Dynamic Help provides just-in-time help that allows you to stay focused on your work losing your
+place typing on the command line.
+
+## Getting cmdlet help
+
+Dynamic Help provides a view of full cmdlet help shown in an alternative screen buffer.
+**PSReadLine** maps the function `ShowCommandHelp` to the <kbd>F1</kbd>key.
+
+- When the cursor is at the end of a fully expanded cmdlet name, pressing <kbd>F1</kbd>displays the
+  help for that cmdlet.
+- When the cursor is at the end of a fully expanded parameter name, pressing <kbd>F1</kbd>displays
+  the help for the cmdlet beginning at the parameter.
+
+![Full screen Dynamic help](./media/dynamic-help/dynamic-help.gif)
+
+The pager in **PSReadLine** allows you to scroll the displayed help using the up and down arrow
+keys. Pressing <kbd>Q</kbd> exits the alternative screen buffer and returns to the current cursor
+position on the command line on the primary screen.
+
+## Focused parameter help
+
+Pressing <kbd>Alt</kbd>-<kbd>h</kbd> provides dynamic help for parameters. The help is shown below
+the current command line similar to `MenuComplete`. The cursor must be at the end of the fully
```

Comment:
May be useful to link docs to `MenuComplete` as I often forget it is a feature and did not connect the similarities without googling it

Reply:
I will link this to the menu complete example (with gif) in the other article.

Reply:
Done